planner/source/docs/mission-planner-configuration-and-tuning.rst

```diff
@@ -1,105 +1,101 @@
 .. _mission-planner-configuration-and-tuning:
 
 ========================================
-Mission Planner Configuration and Tuning
+Mission Planner CONFiguration and Tuning
```

Comment:
any reason for the CONF parts to be capitalised?

Reply:
Matches the screen menu item name...not set on it....thought it might be less confusing to a newbie